services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.scss

```diff
@@ -1,57 +1,60 @@
 @use '@material/typography/mdc-typography';
 @use '@material/card/mdc-card';
 
-.release-dialog-commitId {
-    border: 1px solid white;
+.release-dialog {
+    border-radius: 15px 15px 0px 0px;
+    background-color: transparent;
+    border-radius: 10px;
+}
+
+.release-dialog div.MuiPaper-root {
+    background-color: transparent;
     border-radius: 15px;
 }
 
-.release-dialog {
-    border-radius: 25px 25px 0px 0px;
+.release-env-list {
+    background-color: var(--mdc-theme-background);
 }
 
 .release-dialog-app-bar {
-    border-radius: 25px 25px 0px 0px;
+    border-radius: 10px 10px 0px 0px;
     background: var(--mdc-theme-primary);
     color: var(--mdc-theme-on-primary);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
 }
 
-.release-dialog div div {
-    border-radius: 25px 25px 0px 0px;
-}
-
-.release-dialog-message {
-    margin-top: 15px;
-    margin-left: 30px;
-    font-weight: bold;
-    font-size: medium;
+.release-dialog-app-bar-data {
+    margin: 24px 20px 24px 47px;
     display: flex;
+    flex-direction: column;
     justify-content: space-between;
 }
 
-
-.release-dialog-close {
-    align-items: right;
+.release-dialog-message {
+    @extend .sub-headline1;
     display: flex;
+    justify-content: space-between;
 }
 
 .release-dialog-commitId {
-    margin-left: 20px;
-    border: 1px solid white;
-    border-radius: 8px;
-    font-weight: normal;
-    padding: 5px;
-    display: flex;
+    height: 28px;
+    width: 90px;
```

Comment:
This is small for displaying `undeploy version` please increase it to `120px`

Reply:
made it adaptable:
it now has width set to `auto` and a bit of padding so it wraps nicely around the content ('undeploy version' or the commit hash')